src/main/java/org/sagebionetworks/bridge/sdk/BridgeResearcherClient.java

```diff
@@ -17,43 +18,49 @@
     }
 
     @Override
-    public ResourceList<StudyConsent> getAllStudyConsents() {
+    public ResourceList<StudyConsent> getAllStudyConsents(SubpopulationGuid subpopGuid) {
```

Comment:
Is there *always* a subpopulation GUID? Even if the study only has one population?